addons/tmi/tmi.gd

```diff
@@ -70,14 +71,14 @@ func _ready():
 	add_child(irc)
 	add_child(eventsub)
 	
-	var prev_connection_state = ConnectionStatus.NOT_CONNECTED
+	var _prev_connection_state = ConnectionStatus.NOT_CONNECTED
```

Comment:
These changes do not appear to be necessary, so reverting them would be preferred.